src/de/jost_net/JVerein/gui/navigation/MyExtension.java

```diff
@@ -338,6 +332,9 @@ public void extend(Extendable extendable)
           null);
       einstellungenerweitert.addChild(new MyItem(einstellungenerweitert, "Migration",
           new MitgliedMigrationAction(), "file-import.png"));
+      einstellungenerweitert
+      .addChild(new MyItem(einstellungenerweitert, "QIF Datei-Import",
+          new QIFBuchungsImportViewAction(), "file-import.png"));
```

Comment:
Ich hätte eine kleine Bitte. Du hast QIF und Migration nach Erweitert verschoben. Würdest du auch in DokumentationUtil die Links in den Erweitert Bereich verschieben und den Link nach ADMERWEITERT ändern. Ich würde dann nach Release Freigabe die Datei in der Online Help verschieben.
Ich habe in letzter Zeit die Doku auf Vordermann gebracht und die Verzeichnisstruktur an den Menübaum angepasst. So findet man sich besser in der Doku zurecht.

Ich werde auch nach Release Freigabe alle Screen Shots neu machen damit sie das neue GUI wiederspigeln. Für das Anlagenkonto muss ich ja auch eine gute Beschreibung machen.

Reply:
Habe ich gemacht.
Super das du dich um die Doku kümmerst!